examples/ios_app/test/fixtures/bwb.xcodeproj/xcshareddata/xcschemes/ExampleTests.xcscheme

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I just noticed this, but the scheme name should be the target name, not the product name. So this one should be `ExampleTests.__internal__.__test_bundle` not `ExampleTests`.

Reply:
Done.